conf/application.conf

```diff
@@ -193,7 +193,7 @@ feedback {
   url = "http://localhost:9514/feedback/FUS"
 }
 
-accessibility-link-url = "/cds-file-upload-service/help/accessibility"
+accessibility-statement.service-path = "/cds-file-upload"
```

Comment:
I am still not convinced about this place in the config. But I guess it is already being used in different environment configurations.

Reply:
This is how we need to add the config. The Accessibility Statement service expects that config there.
And it does not work seamlessly in local dev as well, as is normally the case with these `play-ui` integrations.
I would have happily put it on another structure or make it running locally, but if we want to follow the service integration this is it. we cannot change.

Reply:
Even trying to overrride that service-path with a protocol, domain and port, does not work, as they will just append that at the end: 
This is how we check our statement locally: https://github.com/hmrc/cds-file-upload-frontend/pull/95/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R61